<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="192.168.3.2" />
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>
  
  <rosparam command="load" file="$(find panda_controllers)/config/panda_controllers.yaml" />
  <node name="dq_trajectory" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="dq_trajectory"/>
</launch>